### DISCUSSION TITLE
Skycover values differ than what I see on the website

### DISCUSSION THREAD

**Discussion by norin-radd30** [weather-gov/api#620] [Q&A]
Hey I am trying to use the api to get the skycover hourly forecast, here is the api url:
https://api.weather.gov/gridpoints/PDT/23,40

Here is the webpage I am comparing it to:
https://forecast.weather.gov/MapClick.php?w3u=1&w4=sky&w13u=0&w16u=1&AheadHour=0&Submit=Submit&FcstType=digital&textField1=43.9856&textField2=-121.6767&site=all&unit=0&dd=&bw=

The api values differ from what I am seeing on the webpage, and the api is not hourly like the webpage. Which one is accurate? Is there a way to get the api to mimic the webpage in terms of showing the hourly forecast? not just sporadic hour time stamps?

**Comment by dwhitemv25** [comment]
If you are seeing "sporadic" timestamps, make sure you are processing the duration attached to each timestamp (after the '/'). This webpage discusses how to interpret the `/gridpoints` data:

https://weather-gov.github.io/api/gridpoints

You may need to expand the durations into hourly timestamp+forecast for it to make sense to your software.

Both `MapClick.php` and `/gridpoints` use the same data source (NDFD) so they should be equivalent, assuming you are referencing the same gridpoint and there isn't any update lag between the data sources. It might also be that a request for a point forecast for a defined point (like Mt. Bachelor) might have some additional massaging going on compared to the NDFD grids. Looking at my gridpoint right now the values vary by 1 point for Sky Cover between `/gridpoints` and `MapClick`.

**Reply by norin-radd30** [reply]
I read up on the link you sent, and I am not sure it really answered my question as to why the api shows 3-1 hour intervals and the webpage shows 1 hour intervals.  I understand the "PT3H" part describes how long that forecast is good for, but why doesn't it just show every hour like the webpage? Both sources show completely different data. At the time of posting this question the api was showing the sky cover at 65% while the webpage was showing it at 9%....

**Reply by sullynole** [reply]
api times are in UTC so will need to be converted to local time to match web site.  api values are grouped into periods of the same values, even though website breaks those values up into hourly values.  spot check on your links show they match up.

**Reply by norin-radd30** [reply]
Great answer and explanation! thank you, it all matches now!! Much appreciated!!
